
# mapfile: mozambique.yaml #facultatif

# Simulation parameters
use_sim_time: false
trigger_shutdown: 1
coast_margin: 10

# Environment parameters
F_current: 0.0
direction_current: 0.0 #in deg
F_wind: 0.0
direction_wind: 0.0 #in deg
there_are_waves: false

asv:
  heading: 0.0
  u_d: 5.0 #m.s-1
  t_sim: 30

obstacles:
  ship1:
    prior: [stand_on, give_way, none]
    size: [8.0, 30.0] #radius in meters
    heading: [270, 90]
    u_d: [5.0, 10.0] #m.s-1
    t_collision: 15
    d_detection: [500.0, 500.0]
    dcpa: [10.0, 10.0]
